### PR TITLE
ci: daily vm check report

### PR DESCRIPTION
**Pull request by liya2017**
Add a workflow to check vm states daily.

### CODE REVIEW

File: shell-script/useless-vm-check.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            ;;
+        *)
+            SED=sed
+            GREP=grep
+            ;;
+    esac
+    echo "**Daily VM check List**:"
+    echo "| Tag | Instance_id | state |"
+    echo "| :--- | :--- | :--- |"
+    cat ${GITHUB_WORKSPACE}/instances.txt
```
Comment (keroro520) [inline shell-script/useless-vm-check.sh:34]: Can it add more detail about the resources? E.g. region, created-time, resource link.

File: shell-script/useless-vm-check.sh [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+}
+function main() {
+  aws configure set aws_access_key_id ${AWS_ACCESS_KEY}
+  aws configure set aws_secret_access_key ${AWS_SECRET_KEY}
+  aws configure set region us-west-2
+  aws ec2 describe-regions --output text --query Regions[*].[RegionName] >>regions.txt
+  while read -r region;
+  do
+    region=$(echo "$region" | tr -d '[:space:]')
+    aws ec2 describe-instances --region $region |jq '.Reservations[].Instances[] | "| \(.Tags[].Value) | \(.InstanceId) | \(.State.Name) |"'| sed 's/\"//g' >>${GITHUB_WORKSPACE}/instances.txt
```
Comment (keroro520) [inline shell-script/useless-vm-check.sh]: Can it also list other kinds of resources like VPC, keypair, sg? (Check AWS resources but not only EC2)

**Review by keroro520 — CHANGES_REQUESTED** [review]

File: shell-script/useless-vm-check.sh (hunk 2 above)
Comment (liya2017) [reply shell-script/useless-vm-check.sh]: sure，will have a try.

File: shell-script/useless-vm-check.sh (hunk 1 above)
Comment (liya2017) [reply shell-script/useless-vm-check.sh:34]: done

**Review by keroro520 — APPROVED** [review]

File: shell-script/useless-vm-check.sh [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+            SED=gsed
+            GREP=ggrep
+            ;;
+        *)
+            SED=sed
+            GREP=grep
+            ;;
+    esac
+    echo "**Daily VM check List**:"
+    echo "| Tag | Instance_id | State | Region | LaunchTime | Instance_Link |"
```
Comment (keroro520) [inline shell-script/useless-vm-check.sh]: Inconsistent spelling.

**Review by keroro520 — APPROVED** [review]

**Pull request merged**